packages/xeus/src/worker.ts

```diff
@@ -118,23 +118,38 @@ export class XeusRemoteKernel {
     });
     try {
       await waitRunDependency();
-
+      if (globalThis.Module.FS !== undefined) {
```

Comment:
Maybe we should also check that we can actually load dynamic libraries:

```suggestion
      if (globalThis.Module.FS !== undefined && globalThis.Module.loadDynamicLibrary !== undefined) {
```